### PR TITLE
Expose electionChecker to a http server

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

**Comment by Poor12** [comment]
kindly ping @fisherxu.

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ func StartHTTPServer() {
 	router := mux.NewRouter()
 	router.HandleFunc("/edge.crt", edgeCoreClientCert).Methods("GET")
 	router.HandleFunc("/ca.crt", getCA).Methods("GET")
+	router.HandleFunc("/isCloudcoreReady", electionHandler).Methods("GET")
```
Comment (daixiang0) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:48]: Could we simpify this endpoint as `/ready`?

Comment (Poor12) [reply cloud/pkg/cloudhub/servers/httpserver/server.go]: OK.Changed.

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 2]
```diff
@@ -45,6 +45,7 @@ func StartHTTPServer() {
 	router := mux.NewRouter()
 	router.HandleFunc("/edge.crt", edgeCoreClientCert).Methods("GET")
 	router.HandleFunc("/ca.crt", getCA).Methods("GET")
+	router.HandleFunc("/ready", electionHandler).Methods("GET")
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:48]:
```suggestion
	router.HandleFunc("/readz", electionHandler).Methods("GET")
```

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 3]
```diff
@@ -45,6 +45,7 @@ func StartHTTPServer() {
 	router := mux.NewRouter()
 	router.HandleFunc("/edge.crt", edgeCoreClientCert).Methods("GET")
 	router.HandleFunc("/ca.crt", getCA).Methods("GET")
+	router.HandleFunc("/readz", electionHandler).Methods("GET")
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:48]:
```suggestion
	router.HandleFunc("/readyz", electionHandler).Methods("GET")
```

Comment (fisherxu) [reply cloud/pkg/cloudhub/servers/httpserver/server.go:48]: @Poor12  Please fix the suggested change and squash the commits :)

Comment (Poor12) [reply cloud/pkg/cloudhub/servers/httpserver/server.go]: done

**Pull request merged**